src/azure/Model/MethodPya.cs

```diff
@@ -110,7 +110,7 @@ public override string ReturnEmptyResponse
                     if (this.Responses[code].Headers != null)
                     {
                         builder.AppendLine("client_raw_response.add_headers({").Indent();
-                        AddHeaderDictionary(builder, (CompositeType)this.Responses[code].Headers);
+                        AddHeaderDictionary(builder, (Core.Model.CompositeType)this.Responses[code].Headers);
```

Comment:
Why do you need this? It seems the only change in this file, so I guess not really necessary?